lib/requestwrapper.js

```diff
@@ -153,7 +153,7 @@ function createRequest(parameters, _callback) {
   // Headers
   options.headers = extend({}, options.headers);
 
-  if (!isBrowser) {
+  if (!isBrowser && !options.headers['User-Agent']) {
```

Comment:
We should add the `User-Agent` that the user sends us but not replace the one we are using.